### ISSUE TITLE
TypeError: monaco config error

### ISSUE THREAD

**Issue by lywTYH** [SurenAt93/monaco-react#41]
please add monaca config type.

```
declare namespace monaco {
  function init(): Promise<Monaco>;
}
```
 => 
```
declare namespace monaco {
  function init(): Promise<Monaco>;
  function config(params:{urls:{monacoLoader?:string,monacoBase?:string}} ):void;
}
```

**Comment by SurenAt93** [comment]
Hello @lywTYH, I saw your PR. I am going to merge it, thank you for your support.

**Comment by SurenAt93** [comment]
@lywTYH the new version has already published -> 2.4.0
